docs/design/iam.md

```diff
@@ -126,3 +126,31 @@ Here attached a diagram with all the accounts that we have in our system:
 - IAM UpdateAccessKey: AccessKeyId, Status, UserName
 - IAM DeleteAccessKey: AccessKeyId, UserName
 - IAM ListAccessKeys: UserName (not supported: Marker, MaxItems)
+
+## Other
+### Terminology - AWS vs NooBaa
+|   | AWS | NooBaa |
+|---|-----|--------|
+|   | root account  | account  |
+|   | IAM user  | user  |
+
+#### Root Account / Account
+- Since we understand that NC (NooBaa Non-Containerized) deployment (a lightweight deployment of NooBaa on Linux) the term "root" is associated with Linux root permission, we decided that we will use the term "account" instead of "root account" in our use.
+  - The account is the owner of the users that he created using the IAM API. The account owns the users he owns and manage them (can create, read, update, delete or list them).
+  - The account is the owner of the buckets that he or his IAM users created.
+- In AWS root accounts are only created in the console.  
+While in NooBaa, accounts can be created using the CLI or by using the IAM API CreateUser by account that has the flag `--iam_operate_on_root_account`. An account that has this flag operates the IAM API on accounts instead of users, but it does not own them.
```

Comment:
```suggestion
While in NooBaa, accounts can be created by - 
1. NooBaa CLI `account add` command.
2. IAM API CreateUser operation. The requesting account must have the `iam_operate_on_root_account` property set to true. An account that has `iam_operate_on_root_account` property set to true, will operate on accounts instead of users when calling the IAM API, although it does not own them.
```

Reply:
Changed, thanks.